### ISSUE TITLE
Changing splash screen

### ISSUE THREAD

**Issue by heysadboy** [fossasia/open-event-android#1101]
Currently splash screen looks like this

![screenshot_2017-02-13-19-45-46](https://cloud.githubusercontent.com/assets/22588035/22895154/89d4d348-f241-11e6-992e-a8e4e1cbebfb.png)

I changed it to look like this, nothing wild

![screenshot_2017-02-13-22-59-45](https://cloud.githubusercontent.com/assets/22588035/22895208/ae0ccaf4-f241-11e6-9428-9a0e23ac8ae7.png)

Yes, I would like to work on this.

**Comment by shreyasnbhat** [comment]
Already done in my Pull Request #1075

**Comment by heysadboy** [comment]
@shreyasnbhat Hi. I guess you pull request was to center the Tabs in viewpager as it was there in pull request title, here we follow one pull request per issue, so it would be great if you could ask mentor. Thanks.

**Comment by shreyasnbhat** [comment]
This would be better. :)
The logo must be bigger than that.
Just Check the Issue #1058 
I had posted a suggestion.

**Comment by heysadboy** [comment]
What ever you and mentors feel like doing, I'm okay.

**Comment by the-dagger** [comment]
Hey @heysadboy  and @shreyasnbhat .
Ideally we suggest everyone to ask before starting to work on an issue which is already open.
Also there should be 1 : 1 mapping between issues and pull requests.
Hence, there can only be one issue fixed in a Pull Request (most of the times)

Hope that makes sense :)
